src/shared-components/alert/index.tsx

```diff
@@ -46,30 +59,35 @@ class Alert extends React.Component {
     duration: 3,
     truncateText: false,
     type: 'default',
-    onExit: () => undefined,
+    onExit: (): void => undefined,
   };
 
-  constructor(props) {
-    super(props);
-    this.contentText = React.createRef();
-  }
+  static Container = ({ children }: childrenProp): JSX.Element => (
+    <AlertsContainer>{children}</AlertsContainer>
+  );
+
+  contentText = React.createRef<HTMLDivElement>();
```

Comment:
removed constructor in favor of property initialization

Reply:
This is better anyway since we should've been consistent and setting `state` in the constructor as well 😅